nodeserver/controllers/routes.js

```diff
@@ -0,0 +1,81 @@
+const fs = require('fs');
+const path = require('path');
+const http = require('http');
+const bodyParser = require('body-parser');
+
+module.exports = (server) => {
+
+  server.use(bodyParser.urlencoded( { extended: true } ));
+
+  /**
+   * Controller that routes requests based on the URL using ExpressJS.
+   * Server runs on PORT 3000.
+   **/
+
+  // The root of the project is index.html, which is opened when the project is opened.
+  server.get('/', (request, response) => {
+    response.sendFile(path.resolve('./public/index.html'));
+  });
+
+  // This API call can be used to get the current data.
+  // Reads data from JSON file and returns it.
+  // Using a JSON file since the data is small. Can be extended to DB.
```

Comment:
I don't think theres any rule about comments but I believe if it is a continuous thought and 3 lines or longer you should use a block comment.